File: typescript-client-sdk/src/sdk/models/operations/getexternalplansplanid.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";


export class GetExternalPlansPlanIdPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=external_plan_id" })
  externalPlanId: string;
}

export class GetExternalPlansPlanIdRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: GetExternalPlansPlanIdPathParams;
}

export class GetExternalPlansPlanIdResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;
}

File: go-client-sdk/pkg/models/operations/getexternalplansplanid.go
package operations

type GetExternalPlansPlanIDPathParams struct {
	ExternalPlanID string `pathParam:"style=simple,explode=false,name=external_plan_id"`
}

type GetExternalPlansPlanIDRequest struct {
	PathParams GetExternalPlansPlanIDPathParams
}

type GetExternalPlansPlanIDResponse struct {
	ContentType string
	StatusCode  int
}
